account/routes.js

```diff
@@ -52,26 +53,18 @@ router.get('/signed-in', async (req, res, next) => {
             throw err;
         }
 
-        // Get redirectUri
-        const redirectUri = `${req.protocol}://${req.get('host')}/account/signed-in`;
-
         // Get UserIdToken
         const signInService = createSignInService();
-        const options = {
-            code: queryParams.code,
-            redirectUri
-        };
-        const userIdToken = await signInService.getIdToken(options);
+        const authorisationCode = queryParams.code;
+        const userIdToken = await signInService.getIdToken(authorisationCode);
```

Comment:
I think you were right to rename this function in your original PR, on reflection the routes shouldn't care about Tokens, and it's actually returning the ID anyway, not the token.